src/style.css

```diff
@@ -1 +1,56 @@
-/* Styles go here */
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  background: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  font-family: 'Roboto', sans-serif;
+  height: 60px;
+}
+
+.nav__list {
+  display: flex;
+  margin: 0;
+  padding: 0;
+}
+
+.nav__item {
+  list-style: none;
+  margin-left: 20px;
+}
+
+.nav__item:nth-child(1) {
```

Comment:
`:nth-child(1)` is `:first-child`. try to set `margin-left` with `:not(:first-child)`